flake.nix

```diff
@@ -41,14 +69,17 @@
           '';
         };
       };
+      apps.clang-lint = {
+        type = "app";
+        program = getExe clang-lint;
+      };
```

Comment:
Perhaps this part could be added to formatter. You can change it to a script calling both alejandra and clang-lint

Reply:
That's a cool idea. I noticed `clang-tidy -fix` did make changes that broke code and is a little slower than your average formatter. We could just put `clang-format` and `alejandra` together as the formatter, but it is quite neat having clang-tidy and clang-format called from the same script.